source/agora/test/Base.d

```diff
@@ -1331,6 +1331,10 @@ public class TestNetworkManager : NetworkManager
         Duration timeout)
     {
         auto tid = this.registry.locate!TestAPI(address.host);
+        if (tid != typeof(tid).init)
+            return new RemoteAPI!TestAPI(tid, timeout);
+        // Try again
+        tid = this.registry.locate!TestAPI(address.host);
```

Comment:
Separated commit, maybe

Reply:
I'll remove as I think there is another solution pending in a different PR.